app/assets/stylesheets/shareable.scss

```diff
@@ -0,0 +1,10 @@
+nav.share {display:inline-block;} /* shareable's nav tag */
+nav.share div {float:left;}
```

Comment:
Merge rule `nav.share div` with rule on line 1<br>Avoid qualifying class selectors with an element.<br>Colon after property should be followed by one space